src/integration/transaction/application/repositories/transaction.repository.ts

```diff
@@ -16,4 +22,8 @@ export class TransactionRepository extends Repository<Transaction> {
       where: { issuerSignature: Not(IsNull()), verifierSignature: Not(IsNull()), signedHex: IsNull() },
     });
   }
+
+  private oneHourInPast(): string {
+    return new Date(Date.now() - 1 * 60 * 60 * 1000).toISOString();
```

Comment:
Maybe this?
```ts
const date = new Date();
date.setHours(date.getHours() - 1);
return date;
```
Or add a util method next to `daysBefore` (`util.ts`).

Reply:
And the `toISOString` can be omitted.

Reply:
without `toISOString` I had problems during my test, that it works correctly. I agree that it should be moved to utils

Reply:
Hm interesting. I think we used this quite often as a date.

Reply:
Maybe my usage with `LessThan` is wrong or has any implications on it?